src/store.ts

```diff
@@ -87,10 +99,18 @@ function reducer(
 				...state,
 				consoleInteractive: true
 			};
+		case ActionTypes.INITIALIZE_INVENTORY:
+			return {
+				...state,
+				items: action.value
+			};
 		case ActionTypes.ADD_TO_INVENTORY:
+			const item = state.items[action.value];
```

Comment:
Can you do a check here where if `item` ends up being `undefined` you just return `state` without making any changes